src/core/credentials/transport/tls/grpc_tls_certificate_provider.h

```diff
@@ -214,17 +174,16 @@ class FileWatcherCertificateProvider final
 // thread-safe manner.
 class InMemoryCertificateProvider final : public grpc_tls_certificate_provider {
  public:
-  static std::shared_ptr<InMemoryCertificateProvider> CreateCertificateProvider(
-      std::string root_certificates, PemKeyCertPairList pem_key_cert_pairs);
-  InMemoryCertificateProvider(std::string root_certificates,
-                              PemKeyCertPairList pem_key_cert_pairs);
+  InMemoryCertificateProvider();
   InMemoryCertificateProvider(const InMemoryCertificateProvider&) = delete;
   InMemoryCertificateProvider(InMemoryCertificateProvider&&) = delete;
   InMemoryCertificateProvider& operator=(const InMemoryCertificateProvider&) =
       delete;
   InMemoryCertificateProvider& operator=(InMemoryCertificateProvider&&) =
       delete;
 
+  // TODO(anasalazar): Expose APIs so that callers can update in the root
```

Comment:
nit: I think this is done? There's a broad `ForceUpdate`

Reply:
I asked for this TODO, but the way it's worded is very unclear -- I actually had trouble figuring out what it was saying. :)

I suggest changing it to something like this:

"""
TODO(anasalazar): Expose some API for SpiffeBundleMap, so that callers can pass in the root certs as std::shared_ptr<RootCertInfo>.
"""